src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -185,9 +189,7 @@ public function open()
 	    @stream_set_blocking($this->_socket, 0);
 	    $this->_currentProcessingMessage = '';
 	    //register_tick_function(array($this, 'process'));
-	    if ($this->_logger->isDebugEnabled()) {
```

Comment:
The check is done at this level so we can avoid (if possible) the string concatenation and all the work that implies